.github/file-paths.yaml

```diff
@@ -25,7 +25,7 @@ frontend_sources: &frontend_sources
 
 frontend_specs: &frontend_specs
   - *shared_specs
-  - "frontend/test/!(__support__|__runner__)/**"
+  - "frontend/test/!(__support__/e2e|__runner__/*cypress*|metabase/scenarios|snapshot-creators|metabase-visual)/**"
```

Comment:
1. `__support__` was wrong because it also excluded files that are absolutely needed for frontend specs, like mocks
    - I've scoped it to `__support__/e2e`
2. `__runner__` path also includes some potentially useful files for frontend specs
    - I've scoped it to the [path that we already have for e2e tests `__runner__/*cypress*`](https://github.com/metabase/metabase/blob/master/.github/file-paths.yaml#L73)
3. Changing any e2e spec would trigger FE tests prior to this change
    - I've excluded `frontend/test/metabase/scenarios/` and `frontend/test/metabase-visual`
4. Changing any of the snapshot creators needed for Cypress would trigger all FE tests prior to this change
    - I've exluded `snapshot-creators` folder

Reply:
I used the tool you mentioned but [doesn't seem to match the path we want](https://www.digitalocean.com/community/tools/glob?comments=true&glob=frontend%2Ftest%2F%21%28__support__%2Fe2e%7C__runner__%2F%2Acypress%2A%7Cmetabase%2Fscenarios%7Csnapshot-creators%7Cmetabase-visual%29%2F%2A%2A&matches=false&options=dot%3Atrue&tests=%2F%2F%20Should%20be%20included&tests=frontend%2Ftest%2F.eslintrc&tests=frontend%2Ftest%2Fmetabase%2Fquery_builder%2Fselectors.unit.spec.js&tests=frontend%2Ftest%2Fmetabase%2Fnav%2FProfileLink.unit.spec.js&tests=frontend%2Ftest%2F__support__%2Fmocks.js&tests=frontend%2Ftest%2F__runner__%2Fempty.db.mv.db&tests=frontend%2Ftest%2F__runner__%2Flog4j2.xml&tests=%2F%2F%20Should%20be%20ignored&tests=frontend%2Ftest%2F__support__%2Fe2e%2Fcypress_data.js&tests=frontend%2Ftest%2F__support__%2Fe2e%2Fcommands%2Fdatabase%2FaddH2SampleDatabase.js&tests=frontend%2Ftest%2F__runner__%2Frun_cypress_tests.js&tests=frontend%2Ftest%2F__runner__%2Fcypress-runner-get-version.js&tests=frontend%2Ftest%2Fsnapshot-creators%2Fdefault.cy.snap.js&tests=frontend%2Ftest%2Fmetabase%2Fscenarios%2Fnative%2Fnative-mongo.cy.spec.js&tests=frontend%2Ftest%2Fmetabase%2Fscenarios%2Fnative-filters%2Fhelpers%2Fe2e-field-filter-data-objects.js&tests=frontend%2Ftest%2Fmetabase%2Fscenarios%2Fdashboard-filters%2Fshared%2Fdashboard-filters-sql-date.js&tests=frontend%2Ftest%2Fmetabase%2Fscenarios%2Fdashboard-filters%2Freproductions%2F19494-wrong-default-value-multiple-cards-same-question.cy.spec.js&tests=frontend%2Ftest%2Fmetabase-visual%2Fstatic-visualizations%2Frow.cy.spec.js).
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/1937582/220343963-220fc2ee-bdc4-42bc-90ad-b9bf71325367.png">

Reply:
Hence why the PR is in a draft mode.

The glob is invalid